addons/bestja_estimation_reports/models.py

```diff
@@ -89,6 +89,24 @@ class EstimationReportEntry(models.Model):
         store=True,  # Needed by graph view
         related='estimation_report.responsible_organization',
     )
+    responsible_project = fields.Many2one(
+        'bestja.project',
+        string=u"Projekt super nadrzędny",
```

Comment:
Yes, there are three types of those fields. Top project points to the TOP project, Responsible project points to middle level project in case of organization and middle level project in case of middle level organization, parent project points to parent (for organization -> middle level, for middle level -> top). This is consistent with responsible_organization and allows two types of views in statistics- middle level organizations want to see themselves along with the organizations under and top level organization wants to see only middle level (but with all the lowest level organizations summed up).